packages/suite-desktop/src/libs/power-save-blocker.ts

```diff
@@ -0,0 +1,30 @@
+import { powerSaveBlocker } from 'electron';
+
+export class PowerSaveBlocker {
+    private powerSaveBlockerId: number | undefined;
+    private logger: ILogger;
+
+    constructor() {
+        this.logger = global.logger;
+    }
+
+    startBlockingPowerSave() {
+        if (this.powerSaveBlockerId && this.isBlocking(this.powerSaveBlockerId)) {
+            this.logger.info('power-save-blocker', 'Power save is already blocked');
+            return;
+        }
+        this.logger.info('power-save-blocker', 'Start blocking power save');
+        this.powerSaveBlockerId = powerSaveBlocker.start('prevent-display-sleep');
+    }
+
+    stopBlockingPowerSave() {
+        if (!this.powerSaveBlockerId || !this.isBlocking(this.powerSaveBlockerId)) return;
+        this.logger.info('power-save-blocker', 'Stop blocking power save');
+        powerSaveBlocker.stop(this.powerSaveBlockerId);
```

Comment:
I guess it probably changes with each invocation of `powerSaveBlocker.start` so `powerSaveBlocker.isStarted(id);` will return correct result. but I'd also say that it would feel better to remove if after a succesful stop

Reply:
You are right, event if it will have diff id it makes sense to clear it. So I added it and rebased due to some conflicts.